tox.ini

```diff
@@ -6,13 +6,16 @@ skipsdist = True
 [testenv]
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
+commands = find {toxinidir} -type f -name "*.py[c|o]" -delete
```

Comment:
What is the purpose of this?  We likely need to discuss a better way to do this

Reply:
Basically I'd like to remove those files in case they're around before triggering tests. That's widely used in openstack as well but I'm open to suggestions!